### ISSUE TITLE
Wallet Connect is not working.

### ISSUE THREAD

**Issue by frundik** [MoralisWeb3/demo-apps#61]
In this example (https://ethereum-boilerplate.github.io/ethereum-boilerplate/quickstart), I cannot connect, for example, with Trust Wallet

I scan the QR on my phone, do all the necessary confirmations, but there is no authorization on the site.

I have the same problem on my local server.

**Comment by MyNameIsTakenOMG** [comment]
> In this example (https://ethereum-boilerplate.github.io/ethereum-boilerplate/quickstart), I cannot connect, for example, with Trust Wallet
> 
> I scan the QR on my phone, do all the necessary confirmations, but there is no authorization on the site.
> 
> I have the same problem on my local server.

Yep, same here, and `authenticate({provider:'walletcollect})` just doesn't work as expected in my situation🤦‍♂️
